ast/dml.go

```diff
@@ -1899,6 +1917,47 @@ func (n *ShowStmt) Restore(ctx *RestoreCtx) error {
 		}
 	case ShowProfiles:
 		ctx.WriteKeyWord("PROFILES")
+	case ShowProfile:
+		ctx.WriteKeyWord("PROFILE")
+		if len(n.ShowProfileTypes) > 0 {
+			for i, tp := range n.ShowProfileTypes {
+				ctx.WritePlain(" ")
+				if i != 0 {
+					ctx.WritePlain(",")
+				}
+				switch tp {
+				case ProfileTypeCPU:
+					ctx.WritePlain("CPU")
```

Comment:
```suggestion
					ctx.WriteKeyWord("CPU")
```

change the rest too.

Reply:
yes, should be `WriteKeyWord` here. I'll fix them